### PR TITLE
nixos/tt-rss: small improvements

### LINKED ISSUE #74427: tt-rss should be less verbose by default
**Describe the bug**

tt-rss produces an unreasonable amount of logs by default, on the order of 200 lines of info and lower priority, when refreshing 75 feeds.

**To Reproduce**
1. Enable the tt-rss service

**Expected behavior**
Like most services, only warnings and higher priorities should go into the logs by default.

**Additional context**

It seems tt-rss does not have options to configure this, and based on [this discussion](https://community.tt-rss.org/t/systemd-daemon-is-too-verbose-in-syslog/757) it seems unlikely that there is willingness to change that. But one [solution](https://community.tt-rss.org/t/systemd-daemon-is-too-verbose-in-syslog/757/4) would be to set the following in the service file:

```
StandardOutput=null
StandardError=syslog
```

**Metadata**

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.84, NixOS, 19.09.1434.ca9f709fd0b (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(username): `""`
 - channels(root): `"nixos-19.09.1434.ca9f709fd0b, nixos-unstable-20.03pre200231.7827d3f4497"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: services.tt-rss
# a list of nixos modules affected by the problem
module:
```

### PR DESCRIPTION
**Pull request by symphorien**
* Fix logging by making it quiet by default and setting syslogIdentifier. Fixes #74427
* Restart the daemon on failure

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by symphorien** [comment]
cc tt-rss maintainers @globin @zohl

**Review by ajs124 — APPROVED** [review]

File: nixos/modules/services/web-apps/tt-rss.nix
Comment (aanderse) [inline nixos/modules/services/web-apps/tt-rss.nix:639]: @symphorien see https://github.com/NixOS/nixpkgs/issues/86797#issuecomment-629872567 and let me know what you think :thinking:

Comment (symphorien) [reply nixos/modules/services/web-apps/tt-rss.nix:639]: For tt-rss it is because otherwise it is logged as php, not tt-rss, so this is orthogonal to the naming of start scripts.

Comment (aanderse) [reply nixos/modules/services/web-apps/tt-rss.nix:639]: Ah, yeah that makes sense. Cool, thanks! :+1:

**Comment by aanderse** [comment]
I don't see anyone explicitly saying they have tested this. Since there isn't a NixOS test is anyone able to confirm this works with a real quick run through before we merge?

ping @symphorien @ajs124

**Comment by symphorien** [comment]
Ah I forgot to tick the boxes but yes I tested it, notably the "restart on failure" part.

**Review by aanderse — APPROVED** [review]
Great! I guess we're good to merge then. Thanks :tada:

**Pull request merged**
